test/library/packages/UnitTest/launcher.execopts

```diff
@@ -0,0 +1,5 @@
+test_skip.chpl #launcher.skipfile.good
```

Comment:
I would prefer that we don't use the `launcher.chpl` module as a test itself. This file will someday live within `tools/mason` so won't always be available here. Instead, let's create a wrapper program that uses `launcher.chpl` and executes the main function with its arguments forwarded.

Reply:
I have created a basic wrapper program which calls the `Launcher.main()` with the arguments forwarded.